libs/sdk-core/src/chain.rs

```diff
@@ -75,13 +75,21 @@ impl AddressUtxos {
 }
 
 /// Gets unspent tx outputs. Specifically filters out inbound utxos that have been spent.
-pub(crate) fn get_utxos(address: String, transactions: Vec<OnchainTx>) -> Result<AddressUtxos> {
+/// If include_unconfirmed_spends is true, then the result won't include utxos that were spent
+/// in unconfirmed transactions.
```

Comment:
Actually in case where this flag is set to true,  the result will not include confirmed utxos if they were spend by unconfirmed transactions.
This is because we consider these spends even though they are not confirmed

Reply:
Apologies, following the code further down it makes sense